### PR TITLE
Fixed installation issue and added status metric

### PR DESCRIPTION
**Pull request by yogeshprasad**
This includes:

- Fixed Installation Issue
- Added Health Status metric
- Improved Documentation

### CODE REVIEW

**Comment by yogeshprasad** [comment]
@srinivas-kandula @vikramraman

File: README.md [hunk 1]
```diff
@@ -1,29 +1,90 @@
...
-## Developing
-To better understand contributing to logstash plugins, visit [their article on custom output plugins](https://www.elastic.co/guide/en/logstash/current/_how_to_write_a_logstash_output_plugin.html).
-
-To test a local build of this plugin,
-
-   1. Install logstash. Either from source or from a package manager is fine.
-   1. Install ruby bundler -- `gem install bundler` should do it.
-   1. In this directory, `gem build logstash-output-wavefront.gemspec`
-   1. `logstash-plugin install *wavefront*.gem`
+# Setup
```
Comment (srinivas-kandula) [inline README.md]: Move "Install from source" section to here and name the section as "Installation".

File: README.md [hunk 2]
```diff
@@ -1,29 +1,90 @@
...
+Install Wavefront Logstash Output Plugin
+```
+gem install logstash-output-wavefront
+```
+# Enable Output plugin
+Create a config file that specifies `wavefront` as output plugin and settings for other plugins.
```
Comment (srinivas-kandula) [inline README.md]: Which are the other plugins?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,91 @@
 # Wavefront Logstash Output Plugin
-Fully free and fully open source. The license is Apache 2.0, meaning you are pretty much free to use it however you want.
+Wavefront Output Plugin for Logstash parse the log data and sends it as metrics to the wavefront.
```
Comment (srinivas-kandula) [inline README.md]: Change "wavefront" to "Wavefront"

File: README.md [hunk 4]
```diff
@@ -1,29 +1,91 @@
...
+ }
+```
+Below metrics are the output of the above log data
+```
+logstash.bytes.count 200 type=access region=mumbai
+logstash.bytes.mean 42.2 type=access region=mumbai
+logstash.error.count 123
+```
+
+**Note:** Wavefront output plugin for Logstash has dropped out the `logstash.error.code` metric, as default `metrics` list only includes `count and mean`, to include the `code` metric you have to override the default `metrics` to `["count", "mean", "code"]`.
```
Comment (srinivas-kandula) [inline README.md]: Where is the default metric list and how can user change this?

Comment (yogeshprasad) [reply README.md]: Default metric list I have mentioned in the Optional Configuration section of the Wavefront output plugin.

**Comment by srinivas-kandula** [comment]
Could you create examples directory and add example config file into it.

**Comment by srinivas-kandula** [comment]
LGTM

**Review by vikramraman — APPROVED** [review]

**Pull request merged**